### PR TITLE
Add profiler session templates, and ability to list all existing sessions

### PR DESCRIPTION
**Pull request by madelinemacd**
This PR changes the existing startSession functionality, and adds two new request types, CreateSession and getXEventSessions. It also adds a new notification to SQL Ops when a new session has been successfully created.

### CODE REVIEW

**Comment by anthonydresser** [comment]
Were we planning on removing this from the default protocol code?

**Comment by madelinemacd** [comment]
@anthonydresser To move it into mssql? I know that was discussed, do you think it's important to do that now? I'm not sure if there will be time to get that change done before I finish my internship on Friday

**Comment by anthonydresser** [comment]
Sure I was just wondering.

File: src/types.ts [hunk 1]
```diff
@@ -884,6 +903,23 @@ export interface PauseProfilingParams {
...
+	 * Session Owner URI
+	 */
+	ownerUri: string;
+}
+
+export interface GetXEventSessionsResponse {
+	/**
+	 * List of all running XEvent Sessions on target server
+	 */
+	sessions: string[];
```
Comment (MattIrv) [inline src/types.ts]: it may be better to call this `sessionNames`

**Pull request merged**